cesm2_albedo/README

```diff
@@ -0,0 +1,75 @@
+# Albedo Data from CESM2
+
+## Overview
+
+This artifact contains scripts that process two Community Earth System Model 2 data files
+and creates two output files that contain Albedo data. Below is a detailed description of
+each file and its purpose.
+
+## Usage
+
+To recreate the artifact:
+
+Run `julia --project create_artifact.jl`
+
+## Input Files
+
+### 1. `rsds_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc`
+
+This netCDF file contains Surface Upwelling Shortwave Radiation in the variable `rsus`.
+This variables is defined on the longitude, latitude, and time dimensions. `rsus` is taken as
+the montly average, with the first data point at 15/01/1850 12:00 and the las at 15/12/2014 12:00.
+This data file can be downloaded from [LLNL](https://aims3.llnl.gov/thredds/fileServer/css03_data/CMIP6/CMIP/NCAR/CESM2/historical/r1i1p1f1/Amon/rsds/gn/v20190308/rsds_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc
+)
+
+### 2. `rsus_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc`
+
+This netCDF file contains Surface Downwelling Shortwave Radiation in the variable `rsds`.
+This variables is defined on the longitude, latitude, and time dimensions. `rsds` is taken as
+the montly average, with the first data point at 15/01/1850 12:00 and the las at 15/12/2014 12:00.
+This data file can be downloaded from [LLNL](https://aims3.llnl.gov/thredds/fileServer/css03_data/CMIP6/CMIP/NCAR/CESM2/historical/r1i1p1f1/Amon/rsus/gn/v20190308/rsus_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412.nc
+)
+
+## Output Files
+
+### 1. `sw_albedo_Amon_CESM2_historical_r1i1p1f1_gn_185001-201412_v2_no-nans.nc`
+
+This netCDF file contains shortwave albedo in the variable `sw_alb`. The variable is defined
+on the longitude, latitude, and time dimensions. `sw_alb` is taken as the montly average,
+with the first data point at 15/01/1850 12:00 and the las at 15/12/2014 12:00. `sw_alb` is
+calculated by dividing Surface Upwelling Shortwave Radiation by Surface Downwelling Shortwave Radiation.
+At any point, if Surface Downwelling Shortwave Radiation is zero, `sw_alb` is set to 1.0.
+`sw_alb` is stored as a Float32.
+
+### 2. `bareground_albedo.nc`
+
+This netCDF file contains average shortwave albedo in the summer for each hemisphere.
+The variable is defined on the longitude and latitude. In the northern hemisphere the average
+is taken of shortwave albedo during May, June, and July. In the southern hemisphere the average
+is taken of shortwave albedo during December and January.
+At any point, if data is missing, `sw_alb` is set to 1.0.
+`sw_alb` is stored as a Float32.
```

Comment:
I am not sure how accurate this is, but it seemed like what the original script output